src/libraries/System.Diagnostics.Process/src/System/Diagnostics/Process.cs

```diff
@@ -557,6 +572,7 @@ public ProcessStartInfo StartInfo
         {
             get
             {
+                CheckDisposed();
```

Comment:
Same here, ProcessStartInfo should be returned even if process has been disposed.